package.json

```diff
@@ -13,8 +14,9 @@
                 "@microsoft/applicationinsights-react-js": "3.3.6",
                 "@microsoft/applicationinsights-web": "2.8.6",
                 "@microsoft/microsoft-graph-client": "3.0.2",
+                "@microsoft/oteljs": "file:packages/oteljs-4.8.18.tgz",
                 "@monaco-editor/react": "4.4.5",
-                "@ms-ofb/officebrowserfeedbacknpm": "file:packages/officebrowserfeedbacknpm-1.6.6.tgz",
+                "@ms-ofb/officebrowserfeedbacknpm": "file:packages/officebrowserfeedbacknpm-2.0.0.tgz",
```

Comment:
Is this change related to this PR?

Reply:
Oh no! It doesn't. My bad. Let me remove that.